src/components/TestToolMenu/BaseTestToolMenu.tsx

```diff
@@ -88,16 +88,18 @@ function TestToolMenu({user = USER_DEFAULT, network}: TestToolMenuProps) {
                     onPress={() => Session.invalidateCredentials()}
                 />
             </TestToolRow>
+
+            {children}
```

Comment:
why modify this instead of just conditionally the component from https://github.com/Expensify/App/blob/824393109eb1f7f49b9f38f0ca97c8dd93052191/src/components/TestToolMenu.tsx

Reply:
I think we can still do that by creating a component and rendering it here: https://github.com/Expensify/App/blob/824393109eb1f7f49b9f38f0ca97c8dd93052191/src/components/TestToolMenu.tsx#L84 like the other menus

Reply:
@ShridharGoel bump on this

Reply:
We can export an empty component from `index.ts` and the actual component from `index.native.ts`?